app/styles/partials/notifications.scss

```diff
@@ -162,3 +162,43 @@ $warning_color: #f44e06;
     opacity: 0;
   }
 }
+
+.notif-i{
+  padding: 20px 30px 34px 19px;
```

Comment:
Why are there so many style definitions for a simple list ? Try to make use of what is already provided by semantic-ui. That way, it's easier for us to theme the site later.
I'm pretty sure these can be implemented without defining these custom styles.

And name your classes properly. `notif` doesn't make sense. Use `notification`